apps/pages/src/utils/generateHtmlPreview.js

```diff
@@ -0,0 +1,51 @@
+export function generateHtmlPreview(instance, content, ref) {
+  if (content) {
```

Comment:
this should be content && Object.keys(content).length

( else { } is also valid )

Reply:
maybe add the original code to the else? so we do not break any existing pages.

Reply:
Good idea. I've added additional checks.